granary/flickr.py

```diff
@@ -511,6 +523,18 @@ def photo_to_activity(self, photo):
         'displayName': tag.strip(),
       } for tag in photo.get('tags').split(' ') if tag.strip()]
 
+    # location is represented differently in a list of photos vs a
+    # single photo info
+    lat = photo.get('latitude') or photo.get('location', {}).get('latitude')
+    lng = photo.get('longitude') or photo.get('location', {}).get('longitude')
+    if lat and lng and float(lat) != 0 and float(lng) != 0:
+      activity['object']['location'] = {
+        'objectType': 'place',
+        'latitude': float(lat),
+        'longitude': float(lng),
+      }
+
+    self.postprocess_object(activity['object'])
```

Comment:
I had that thought too, but it looks right the way it is on the other sources -- most seem to have a clearer delineation between object and activity than Flickr does. Like you have both tweet_to_object and tweet_to_activity (which makes sense bc tweets can nest or reference each other)